Resources/doc/index.rst

```diff
@@ -348,5 +348,21 @@ class (as shown above), you can access it in the ``load()`` method.
     If you are too lazy to implement the needed method :method:`Symfony\\Component\\DependencyInjection\\ContainerInterface::setContainer`,
     you can then extend your class with :class:`Symfony\\Component\\DependencyInjection\\ContainerAware`.
 
+Automatically Generated Fixtures with Hautelook/AliceBundle
+-----------------------------------------------------------
+
```

Comment:
my proposal:

When you just need random mock data to test things, you do not need to write all of that out in fixture files. Instead, you can use alice_ to configure how to generate any amount of entities with random data produced by faker_. This combo is really powerful as it can produce realistically looking data that can even be tuned for certain countries and can be set to generate nasty edge cases.

To integrate alice and faker with Symfony, you can use `Hautelook/AliceBundle`_.


if you want to do an exmaple, show the yaml below and then the command to run. but that would actually belong to the doc of Hautelook/AliceBundle so maybe we should not show an example at all here.